org.elysium.parent/org.elysium.ui/src/org/elysium/ui/refactoring/LilyPondRefactoring.java

```diff
@@ -106,13 +115,30 @@ public Operation getOperation() {
 		return operation;
 	}
 
-	public Set<URI> getRefactoredFiles(){
+	public Set<URI> getRefactoredFilesPlatformURIs(){
 		return platformURItoFileOfRefactorTargets.keySet();
 	}
 
-	public List<IPath> getFilePaths(Set<URI> uris){
+	public Set<URI> getRefactoredFilesPlatformURIS(Set<URI> fileURIsOfIncludes){
```

Comment:
Please fix the case: getRefactoredFilesPlatformURIS -> getRefactoredFilesPlatformURIs.